### PR TITLE
HSS-9295: Update to GD 6.1 SDK & latest FIPS_module

### PR DESCRIPTION
**Pull request by dbanik**
- Update to version 6.1.0.170.

https://hearsaysocial.atlassian.net/browse/HSS-9295

@casperson @bblia

### CODE REVIEW

File: GD/FIPS_module/arm64.sdk/bin/gd_fipsld [hunk 1]
```diff
@@ -59,7 +59,7 @@ CLANG_PATH="clang"
 
 # define current architecture
 CUR_ARCH=`(while [ "x$1" != "x" -a "x$1" != "x-arch" ]; do shift; done; echo $2)`
-[ -n "${CUR_ARCH}" ] || { echo 'no -arch specified'; exit 1; }
+#[ -n "${CUR_ARCH}" ] || { echo 'no -arch specified'; exit 1; }
```
Comment (bblia) [inline GD/FIPS_module/arm64.sdk/bin/gd_fipsld]: should this be committed?

Comment (dbanik) [reply GD/FIPS_module/arm64.sdk/bin/gd_fipsld]: Yes, we don't own that. Its fips loader.

**Comment by bblia** [comment]
lgtm

**Comment by casperson** [comment]
lgtm 🤙

**Pull request merged**